command/snapshot_inspect.go

```diff
@@ -0,0 +1,89 @@
+package command
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"bytes"
```

Comment:
Tiny nit - all the standard packages should be sorted in a clump at the top, and then all the github stuff in another clump.